src/index.ts

```diff
@@ -9,17 +8,12 @@ import { DynamoService, IDynamoService } from './IDynamoService';
 export { IDynamoOperation, IDynamoService, DynamoService, DynamoFactory, DynamoServices };
 
 export interface UseDynamoConfiguration {
-  core?: DynamoDBClientConfig;
-  documentTranslation?: TranslateConfig;
+  core?: DynamoDB.Types.ClientConfiguration;
```

Comment:
Since we don't need the documentTranslation stuff, let's just pass in the configuration without this interface

Reply:
removed